.github/workflows/main.yml

```diff
@@ -12,14 +12,17 @@ jobs:
   build:
     strategy:
       matrix:
-        python: [3.9]
+        python-version: [3.9]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python ${{ matrix.python }}
-        uses: actions/setup-python@v4
+      - name: Setup Conda
+        uses: conda-incubator/setup-miniconda@v3
         with:
-          python-version: ${{ matrix.python }}
+          activate-environment: true
+          python-version: ${{ matrix.python-version }}
+          auto-activate-base: true
```

Comment:
Why does the ``base`` environment need to be activated? This is not the recommended practise

Reply:
The discussion was around not needing the `bash -el {0}` commands needed e.g. here: https://github.com/ome/conda-bioformats2raw/blob/cb5a3d9a1ce9bfe72a5b5a1067964939236805f0/.github/workflows/condabuild.yml#L20